index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Chimezie's Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="container">
+            <header class="head-container">
+                <div class="logo">
+                    <h3>Asouzu Chimezie</h3>
+                </div>
+                <div class="menu">
+                    <img src="/images/Union.png" alt="hamburger icon">
+                </div>
+            </header>
+        <section>
+            <div class="main">
+                <div class="wrapper">
+                    <div class="welcome-note">
+                        <h1>I'm Chimezie Glad to see you!</h1>
```

Comment:
It's fantastic 👌and you can make it look the same as Figma if you add some space to the top and use the same color for the title text. Kindly bring down the content, and update its color to match the original design.

![image](https://user-images.githubusercontent.com/34779928/198121334-871645c1-e48a-4d71-a89f-7517e3cd6e75.png)

Reply:
Thanks for the feedback i have done the necessary adjustments.